cmd/query/app/flags.go

```diff
@@ -140,18 +142,20 @@ func (qOpts *QueryOptions) InitFromViper(v *viper.Viper, logger *zap.Logger) (*Q
 
 // BuildQueryServiceOptions creates a QueryServiceOptions struct with appropriate adjusters and archive config
 func (qOpts *QueryOptions) BuildQueryServiceOptions(storageFactory storage.BaseFactory, logger *zap.Logger) *querysvc.QueryServiceOptions {
-	opts := &querysvc.QueryServiceOptions{}
+	opts := &querysvc.QueryServiceOptions{
+		MaxClockSkewAdjust: qOpts.MaxClockSkewAdjust,
+	}
 	if !opts.InitArchiveStorage(storageFactory, logger) {
 		logger.Info("Archive storage not initialized")
 	}
 
-	opts.Adjuster = adjuster.Sequence(querysvc.StandardAdjusters(qOpts.MaxClockSkewAdjust)...)
```

Comment:
why do we need to change this? I think the big might be in `qOpts.MaxClockSkewAdjust` not being assigned. No other changes are required.

Reply:
Sorry I might have misunderstood your [comment](
https://github.com/jaegertracing/jaeger/pull/6566#discussion_r1924588674) from yesterday then.

From that comment I gathered that the old way of setting the adjuster early is not the way to go anymore? So I moved the `MaxClockSkewAdjust` value into the `QueryServiceOptions` to propagate it. But with propagating it down to where its instantiated there is no need to assign the adjuster from outside anymore.

I can of course restore the previous version where we explicitly assign the adjuster from outside its module so it won't use the default value (as does the other part of the code).